CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Fixed
+- Premium Security - Handle mutli instances ip whitelist refresh.
```

Comment:
`IP Whitelist - Request IP whitelist refresh if an IP looks invalid with the current IP whitelist`